### PR TITLE
Resolve problem with saving Event without organizer

### PR DESCRIPTION
**Pull request by bordoni**
🎫  [TEC-3788]
--- 
https://i.bordoni.me/7KupXLrd

[TEC-3788]: https://theeventscalendar.atlassian.net/browse/TEC-3788

### CODE REVIEW

File: src/Tribe/Editor/Meta.php [hunk 1]
```diff
@@ -99,6 +101,61 @@ public function register() {
...
+		if ( ! function_exists( 'wp_is_json_request' ) || ! wp_is_json_request() ) {
+			return $delete;
+		}
+
+		$meta_keys_to_allow = [
+			'_EventOrganizerID' => true,
+			'_EventVenueID' => true,
+		];
+
+		if ( ! isset( $meta_keys_to_allow[ $meta_key ] ) ) {
```
Comment (Camwyn) [inline src/Tribe/Editor/Meta.php:130]: Question/thought: will `'_EventVenueID' => false,` trigger this? Should it?

File: src/Tribe/Editor/Meta.php [hunk 2]
```diff
@@ -99,6 +101,61 @@ public function register() {
...
+			return $delete;
+		}
+
+		if ( ! empty( $meta_value ) ) {
```
Comment (Camwyn) [inline src/Tribe/Editor/Meta.php:134]: Any reason to no bump this up above setting `$meta_keys_to_allow`?

**Review by Camwyn — APPROVED** [review]
Looks good.

File: src/Tribe/Editor/Meta.php (hunk 1 above)
Comment (bordoni) [reply src/Tribe/Editor/Meta.php:130]: It should, this method is just faster then `in_array`.

**Review by Camwyn — APPROVED** [review]

**Pull request merged**